identifier/src/containers/Login/Login.jsx

```diff
@@ -81,7 +81,13 @@ function Login(props) {
       }
 
       history.replace(`/chooseaccount${history.location.search}${history.location.hash}`);
-      return;
+        return;
+      }
+    if (query && query.login_hint) {
+      if (validator.isEmail(query.login_hint) || validator.isEmail(`${query.login_hint}@example.com`)) {
+        // for login hint user name validation
```

Comment:
I don't understand this comment and its location. Please update or remove. Also comments in this project should be full sentences. They are to provide context and reasoning which are not obvious from the code and those comments are for us humans, so it is worth the effort to make them nice.

Reply:
Updated the comment to get full context of the usage.